docs/website/binding_objc_libs.md

```diff
@@ -840,6 +840,51 @@ An alternative is to mark the original, `NSString`-based, definition as
 the wrap'ed version will still work and call the overriden method.
 
 
+## Binding NSValue NSNumber and NSString to a better type
+
+The [[BindAs]](/guides/ios/advanced_topics/binding_objective-c/binding_types_reference_guide#BindAsAttribute) attribute allows binding `NSNumber` `NSValue` and `NSString`(enums) into more accurate C# types this allows an easy way to create better .NET API.
```

Comment:
Missing comma between `NSValue` and `NSNumber` (twice, both in the header and in the text):

... `NSNumber`, `NSValue` and `NSString` ...